cmd/terminal-to-html/go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/buildkite/terminal-to-html/cmd/terminal-to-html/v3
+
+go 1.12
```

Comment:
That's quite an old Go, but doesn't really matter.

Reply:
I'm not even clear on how you "update" that, `go mod tidy` just deletes everything. Go is exhausting.